src/test/java/com/damian/cryptoportfolio/logic/services/CoinServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package com.damian.cryptoportfolio.logic.services;
+
+import com.damian.cryptoportfolio.data.CoinRepository;
+import com.damian.cryptoportfolio.logic.models.Coin;
+import com.damian.cryptoportfolio.logic.models.User;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class CoinServiceTest {
+
+    @InjectMocks
```

Comment:
This line is not doing anything since you are using  
`MockitoAnnotations.initMocks(this);`
This already initialize the mocks